frontend/src/actions/TableActions.js

```diff
@@ -511,25 +525,38 @@ function updateCollapsedRows({ tableId, rows }) {
       collapsedRows,
       collapsedParentRows,
     } = table;
-    let newCollapsedParentRows = [];
-    let newCollapsedRows = [];
+
+    const hasChangedIds =
```

Comment:
`hasChangedIds` name does not make sense anymore when the param name is called `preserveCollapsedStateToRowIds`

Reply:
yep. Thanks for spotting this